src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/OptionsTests.cs

```diff
@@ -512,6 +512,22 @@ public static void CopyConstructor_NullInput()
             Assert.Contains("options", ex.ToString());
         }
 
+        [Fact]
+        public static void JsonSerializerOptions_Default_MatchesDefaultConstructor()
+        {
+            var options = new JsonSerializerOptions();
+            var optionsSingleton = JsonSerializerOptions.Default;
```

Comment:
```suggestion
            JsonSerializerOptions optionsSingleton = JsonSerializerOptions.Default;
            Assert.Same(JsonSerializerOptions.Default, optionsSingleton);
```

Reply:
Thanks. I think I'll move the `Same` assertion to a separate unit test.